decorators.py

```diff
@@ -0,0 +1,14 @@
+def is_command(description=""):
```

Comment:
Again, I'm not really sure why this needs to be in its own file when there's only one short function.